Archive/Imo/Imo2008Q3.lean

```diff
@@ -61,12 +61,12 @@ theorem p_lemma (p : ℕ) (hpp : Nat.Prime p) (hp_mod_4_eq_1 : p ≡ 1 [MOD 4])
   have hreal₃ : (k : ℝ) ^ 2 + 4 ≥ p := by assumption_mod_cast
   have hreal₅ : (k : ℝ) > 4 := by
     refine lt_of_pow_lt_pow_left 2 k.cast_nonneg ?_
-    linarith only [hreal₂, hreal₃]
+    linear_combination hreal₂ + hreal₃
```

Comment:
I feel neutral about these changes. They are marginally longer, but presumably also marginally clearer to the non-Lean expert: "just add these equations" conveys (slightly) useful information. In other words, I am (personally, though my non-maintainer opinion only counts little) fine with merging these.